interactions/models/discord/guild.py

```diff
@@ -360,7 +360,7 @@ def humans(self) -> List["models.Member"]:
     @property
     def roles(self) -> List["models.Role"]:
         """Returns a list of roles associated with this guild."""
-        return [self._client.cache.get_role(r_id) for r_id in self._role_ids]
+        return sorted([self._client.cache.get_role(r_id) for r_id in self._role_ids], reverse=True)
```

Comment:
```suggestion
        return sorted(self._client.cache.get_role(r_id) for r_id in self._role_ids, reverse=True)
```

Reply:
A generator expression must be wrapped in brackets if not the sole argument